README.md

```diff
@@ -515,6 +516,28 @@ Another alternative is [Chuck](https://github.com/jgilfelt/chuck) which, althoug
 
 [LeakCanary](https://github.com/square/leakcanary) is a library that makes runtime detection and identification of memory leaks a more routine part of your application development process. See the library [wiki](https://github.com/square/leakcanary/wiki) for details on configuration and usage. Just remember to configure only the "no-op" dependency in your release build!
 
+### Set up continuous integration
+
+Continuous integration systems let you automatically build and test your project every time you push in updates to version control. CI also runs static code analysis tools, generates the apk files and distributes them.
+Some popular code analysis tools are:
```

Comment:
For this guide, it's better to say what we recommend rather than list the range of alternatively. Perhaps pick the ones that you have used and recommend.

Reply:
👍